### PR TITLE
State PVC requirements

### PR DESCRIPTION
**Pull request by iranzo**
Indicate PVC requirements

Signed-off-by: Pablo Iranzo Gómez <Pablo.Iranzo@gmail.com>

### CODE REVIEW

File: administration/live-migration.adoc [hunk 1]
```diff
@@ -32,6 +32,8 @@ data:
   feature-gates: "DataVolumes,LiveMigration"
 ....
 
+Additionally, Virtual machines must have a PersistentVolumeClaim (PVC) with a shared ReadWriteMany (RWX) access mode to be live migrated.
```
Comment (vladikr) [inline administration/live-migration.adoc]: @iranzo Thanks for this clarification. However, I think we should further clarify it as having a PVC is not a requirement for live migration - it can have any of the supported disks. Only if the VM uses a PVC it should be RWX.

Comment (iranzo) [reply administration/live-migration.adoc]: ok, let me rephrase

File: administration/live-migration.adoc [hunk 2]
```diff
@@ -32,6 +32,8 @@ data:
   feature-gates: "DataVolumes,LiveMigration"
 ....
 
+Additionally, Virtual machines using a PersistentVolumeClaim (PVC) must have a shared ReadWriteMany (RWX) access mode to be live migrated.
```
Comment (vladikr) [inline administration/live-migration.adoc]: There is a similar comment on line 61, in "Condition and migration method"

File: administration/live-migration.adoc (hunk 1 above)
Comment (vladikr) [reply administration/live-migration.adoc]: Thanks :)

File: administration/live-migration.adoc (hunk 2 above)
Comment (iranzo) [reply administration/live-migration.adoc]: Should we move that one to the upper part to make it clear that is a requirement and not just a 'rejection' ?

Comment (vladikr) [reply administration/live-migration.adoc]: Perhaps we need a new section with "Limitations", as we also won't allow a migration with a bridge interface attached to a pod network...[1]

https://kubevirt.io/user-guide/docs/latest/creating-virtual-machines/interfaces-and-networks.html

Comment (vladikr) [reply administration/live-migration.adoc]: There is also another PR[1] in progress that will add new limitations.

[1] https://github.com/kubevirt/kubevirt/pull/2982

Comment (iranzo) [reply administration/live-migration.adoc]: I'll add mention to it or should we have that other 'on watch' to update when merged?

Comment (vladikr) [reply administration/live-migration.adoc]: I'll add a comment in the PR ^^ to update this section after it's merged.

Comment (iranzo) [reply administration/live-migration.adoc]: Done!
